packages/app/src/cli/models/extensions/specifications/payments_app_extension.test.ts

```diff
@@ -0,0 +1,102 @@
+import {PaymentsAppExtensionConfigType} from './payments_app_extension.js'
+import {OffsitePaymentsAppExtensionConfigType} from './payments_app_extension_schemas/offsite_payments_app_extension_schema.js'
+import {testPaymentsAppExtension} from '../../app/app.test-data.js'
+import {ExtensionInstance} from '../extension-instance.js'
+import * as upload from '../../../services/deploy/upload.js'
+import {loadFSExtensionsSpecifications} from '../load-specifications.js'
+import {inTemporaryDirectory, writeFile} from '@shopify/cli-kit/node/fs'
+import {beforeEach, describe, expect, test, vi} from 'vitest'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+vi.mock('../../../services/deploy/upload.js')
+
+describe('PaymentsAppExtension', () => {
+  let extension: ExtensionInstance<PaymentsAppExtensionConfigType>
+  const moduleId = 'module_id'
+  const apiKey = 'app-key'
+  const token = 'app-token'
+  const inputQuery = 'query { f }'
+
+  const config: OffsitePaymentsAppExtensionConfigType = {
+    name: 'test extension',
+    type: 'payments_extension',
+    payment_session_url: 'http://foo.bar',
+    refund_session_url: 'http://foo.bar',
+    capture_session_url: 'http://foo.bar',
+    void_session_url: 'http://foo.bar',
+    confirmation_callback_url: 'http://foo.bar',
+    merchant_label: 'some-label',
+    supported_countries: ['CA'],
+    supported_payment_methods: ['PAYMENT_METHOD'],
+    supports_3ds: false,
+    supports_oversell_protection: false,
+    test_mode_available: true,
+    supports_deferred_payments: false,
+    supports_installments: false,
+    target: 'offsite',
```

Comment:
This target has been updated to `payments.offsite.render` to conform the existing naming conventions
https://github.com/Shopify/extensions-templates/pull/78/files#diff-16ea8a4546856fe48261c235611ee38ba8f4cf2ac96d9a8e62ea0ce63a680c7eR11